### ISSUE TITLE
Result depends on value of out-variable

### ISSUE THREAD

**Issue by Joel-Dahne** [kalmarek/Arblib.jl#70]
Consider the code below
```julia
x = Acb(Arblib.load_string!(Arb(),"1 -2 20000001 -1f"))
res1 = Acb(Arblib.load_string!(Arb(),"1 -1 8000001 -1c"))
res2 = Acb(Arblib.load_string!(Arb(),"0 0 0 0"))
Arblib.real_abs!(res1, x, false)
Arblib.real_abs!(res2, x, false)
```
You would expect `res1` and `res2` to have the same value, however
```julia
julia> res1
[+/- 1.01]
julia> res2
[+/- 0.501]
```
I encountered this issue when trying to track down some bugs in the integration. So far I have not been able to track down exactly where things go wrong so I thought I would put it here in case you find something before I do.

**Comment by Joel-Dahne** [comment]
Turns out to actually be a bug in Arb https://github.com/fredrik-johansson/arb/pull/350

**Comment by Joel-Dahne** [comment]
Closed by #111
